### PR TITLE
Update Vault guides to the new Testnet

### PR DESCRIPTION
**Pull request by b00ste**
### What does this PR introduce?

The changes update the code to work with the new LUKSO Testnet.
Additionally **Create a Vault**, **Edit Vault Data** & **Interact With Contracts**  have been tested.

**Restrict Addresses to Vaults** guide has to be updated & tested after _@erc725/erc725.js_ repository is updated. The `AllowedCalls` encoding must be updated to the new way.

### CODE REVIEW

File: docs/guides/vault/edit-vault-data.md [hunk 1]
```diff
@@ -424,10 +426,13 @@ await updateVaultURD(vaultURDAddress);
 import LSP1UniversalReceiverDelegateVault from '@lukso/lsp-smart-contracts/artifacts/LSP1UniversalReceiverDelegateVault.json';
 import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
 import LSP9Vault from '@lukso/lsp-smart-contracts/artifacts/LSP9Vault.json';
-import { ERC725YDataKeys } from '@lukso/lsp-smart-contracts/constants.js';
+import {
+  ERC725YDataKeys,
+  SupportedStandards,
```
Comment (CJ42) [inline docs/guides/vault/edit-vault-data.md:431]: this imported value seems to not be used.

File: docs/guides/vault/edit-vault-data.md [hunk 2]
```diff
@@ -451,9 +456,7 @@ const deployVaultURD = async () => {
       gas: 5_000_000,
       gasPrice: '1000000000',
     })
-    .on('receipt', (receipt) => {
-      return receipt.contractAddress;
-    });
+    .on('receipt', (receipt) receipt.contractAddress;);
```
Comment (CJ42) [inline docs/guides/vault/edit-vault-data.md:459]: let's use the arrow syntax as it was.

File: docs/guides/vault/edit-vault-data.md [hunk 3]
```diff
@@ -520,7 +525,7 @@ const deployVaultURD = async () => {
   // deploy the Universal Receiver Delegate Vault contract
   const vaultURD = await vaultURDFactory.connect(myEOA).deploy();
 
-  return vaultURD.target;
+  return vaultURD.address;
```
Comment (CJ42) [inline docs/guides/vault/edit-vault-data.md:528]: good catch 👌

File: docs/guides/vault/restrict-addresses-to-vaults.md [hunk 4]
```diff
@@ -270,6 +272,12 @@ const allowedCallsSchema = {
   valueContent: '(Bytes4,Address,Bytes4)',
 };
 
+const allowedCallsDataValue = encodeKey(allowedCallsSchema, [
```
Comment (CJ42) [inline docs/guides/vault/restrict-addresses-to-vaults.md:275]: good catch 👌

**Review by YamenMerhi — APPROVED** [review]

**Review by CJ42 — APPROVED** [review]

**Pull request merged**